### DISCUSSION TITLE
Feedforward is deprecated? Module needs shapes while binding. How can we do it without shapes?

### DISCUSSION THREAD

**Discussion by ghost** [apache/incubator-mxnet#15074] [Q&A]
I tried mx.mod.Module but it takes arguments such as data shapes when binding.
In feed forward we didn't need data shapes. What if i dont know the data shapes??
How can I incorporate Module?

_Originally posted by @vizakshat in https://github.com/apache/incubator-mxnet/issues/5394#issuecomment-495936386_

**Comment by frankfliu** [comment]
@mxnet-label-bot add [question]

**Comment by chinakook** [comment]
Set some shape first, and it can be changed accord to the given data later.
